finder-app/Makefile

```diff
@@ -0,0 +1,11 @@
+CROSS_COMPILE = gcc
```

Comment:
@TejaswiniLakshminarayan see notes from class video about how to specify CC with CROSS_COMPILE for compatibility with future assignments.